sql/verifier.go

```diff
@@ -72,7 +72,8 @@ func describeTables(slct *extendedSelect, db *DB) (ft fieldTypes, e error) {
 	ft = indexSelectFields(slct)
 	hasStar := len(ft) == 0
 	for _, tn := range slct.tables {
-		q := "SELECT * FROM " + tn + " LIMIT 1"
+		// MaxCompute's statement ends with a semicolon
+		q := "SELECT * FROM " + tn + " LIMIT 1;"
```

Comment:
I vaguely remember Hive client requires SQL statement ending without `;`. Will Hive CI fail due to this change?

Reply:
I tested several extended SQL cases with the hive, it works.

In fact, hive SQL does not need the last `;`, so :
1. Gohive removes the last semicolon, [ref](https://github.com/sql-machine-learning/gohive/blob/develop/connection.go#L83)
2. SQLFlow removes the last semicolon for python db-api, [ref](https://github.com/sql-machine-learning/sqlflow/blob/develop/sql/codegen.go#L135)

Reply:
Great!